onnxruntime/test/providers/cpu/object_detection/non_max_suppression_test.cc

```diff
@@ -344,5 +344,25 @@ TEST(NonMaxSuppressionOpTest, ZeroMaxOutputPerClass) {
   test.Run();
 }
 
+TEST(NonMaxSuppressionOpTest, WithIOUThresholdOpset11) {
+  OpTester test("NonMaxSuppression", 11, kOnnxDomain);
```

Comment:
Add test for opset-11 NMS

Reply:
there's nothing change, they just bump up the opset version?

---
In reply to: [328874393](https://github.com/microsoft/onnxruntime/pull/1941#discussion_r328874393) [](ancestors = 328874393)

Reply:
Long story short - yes, only shape inference was implemented for the new version